vendor/github.com/ulikunitz/xz/TODO.md

```diff
@@ -78,7 +78,7 @@
 * Update README.md
 * xb copyright . in xz directory to ensure all new files have Copyright header
 * `VERSION=<version> go generate github.com/ulikunitz/xz/...` to update version files
-* Execute test for Linux/amd64, Linux/x86 and Windows/amd64.
```

Comment:
Nothing should be changed anywhere in vendor, please remove all these changes. As with composer, these go module changes will just mess things up and be overwritten anyway.